src/simulation_lifecycle/utils/workflow.c

```diff
@@ -0,0 +1,99 @@
+#include <string.h>
+#include <unistd.h>
+#include "cJSON.h"
+#include "simulation_lifecycle/error.h"
+#include "simulation_lifecycle/utils/file.h"
+#include "simulation_lifecycle/utils/workflow.h"
+
+
+cJSON * read_workflow_file(char *path_to_file){
+    cJSON *content = NULL;
+    read_json_file(path_to_file, &content);
+    return(content);
+}
```

Comment:
I wonder if this is necessary since it really only calls the read_json_file function